ui/package.json

```diff
@@ -63,7 +63,6 @@
     "build": "gulp build",
     "fmt": "prettier --write .",
     "gen:browserlist": "gulp gen:browserlist",
```

Comment:
Are we completely removing the Storybook?

Reply:
Yep, it doesn't work anymore. We can bring it back when we really need it, I think.

Reply:
Create an issue #1190 to track it.